cpp/benchmarks/stream_compaction/unique.cpp

```diff
@@ -62,9 +66,12 @@ void nvbench_unique(nvbench::state& state, nvbench::type_list<Type, nvbench::enu
   auto input_column = source_column->view();
   auto input_table  = cudf::table_view({input_column, input_column, input_column, input_column});
 
+  auto const sort_order = cudf::sorted_order(input_table);
```

Comment:
This comment is not for this line but for the line 61 above. The data gen profile should based on the input size. For example, the line 
```
distribution_id::UNIFORM, 0, 100
```
should be replaced by 
```
distribution_id::UNIFORM, 0, num_rows / 100
```

By doing so, we will have more realistic data with approx. constant ratio of duplicates. Otherwise we will have very skewed data. For example, imagine that we have 100 millions rows with values in `[0, 100)`, the duplicate factor is huge and this benchmark may give very strange output.